tests/chainer_tests/functions_tests/activation_tests/test_elu.py

```diff
@@ -0,0 +1,69 @@
+import random
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+class TestELU(unittest.TestCase):
+
+    def setUp(self):
+        # Avoid unstability of numeraical grad
+        self.x = numpy.random.uniform(-1, 1, (5, 4)).astype(numpy.float32)
+        for i in range(self.x.size):
+            if -0.01 < self.x.flat[i] < 0.01:
+                self.x.flat[i] = 0.5
```

Comment:
Unlike ReLU, ELU is differentiable at x = 0. So I do not think we need this avoid sampling around x = 0.